specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/sensitivityLabels.json

```diff
@@ -128,6 +135,126 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/schemas/{schemaName}/tables/{tableName}/columns/{columnName}/sensitivityLabels/recommended/enable": {
```

Comment:
This API signature will not work. ARM expects resourceType/resourceTypeName segments. Here you have a missing segment. What is "recommended"? Is it a resource type or an instance of sensitivityLabels?

Reply:
Recommended sensitivity label, there are two sources of sensitivity labels, either current or recommended, but these actions are applicable only for recommended sensitivity labels, thus it's hardcoded in the URI (You can see the resource id:
/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/schemas/{schemaName}/tables/{tableName}/columns/{columnName}/sensitivityLabels/{sensitivityLabelSource})

Reply:
Got it. Can you make this as an enum and the only allowed value in that would be recommended. It will help you in expanding supported values in future.